src/Take.Blip.Builder/Actions/Redirect/RedirectAction.cs

```diff
@@ -2,26 +2,34 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Newtonsoft.Json.Linq;
+using Serilog;
 
 namespace Take.Blip.Builder.Actions.Redirect
 {
     public class RedirectAction : IAction
     {
         private readonly IRedirectManager _redirectManager;
+        private readonly ILogger _logger;
 
-        public RedirectAction(IRedirectManager redirectManager)
+        public RedirectAction(IRedirectManager redirectManager, ILogger logger)
         {
             _redirectManager = redirectManager;
+            _logger = logger;
         }
 
         public string Type => nameof(Redirect);
 
         public Task ExecuteAsync(IContext context, JObject settings, CancellationToken cancellationToken)
         {
+            var redirect = settings.ToObject<Lime.Messaging.Contents.Redirect>(LimeSerializerContainer.Serializer);
```

Comment:
Can you add a using for `Lime.Messaging.Contents` or is this disambiguation for another `Redirect`?